### PR TITLE
Settings and Sidebar Fully Dark Themed

### PR DESCRIPTION
**Pull request by Crazypkr1099**

### CODE REVIEW

File: upcoming-release-notes/1821.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+category: Enhancements
+authors: Crazypkr
```
Comment (MatissJanis) [inline upcoming-release-notes/1821.md:3]:
```suggestion
authors: [Crazypkr1099]
```

**Review by MatissJanis — COMMENTED** [review]
👋 Overall looks good. I like your dark shade better than the one we had before in the side nav. Though a few small improvements we could make:

1. the separator line has a blue tint - maybe a gray-ish one would be better suited? (red arrow in screenshot)
2. the "Test Budget" also has blue tint - maybe that should be white? (red arrow in screenshot)
3. the hover color of items also ha a blue tint that would be nice to change to something else, but not sure what.. open to ideas :)


| before | after |
|--------|--------|
|Screenshot 2023-10-22 at 16 39 09 | Screenshot 2023-10-22 at 16 38 35
 |

**Comment by MatissJanis** [comment]
Oh also would be good to get @carkom feedback here.

**Comment by carkom** [comment]
First call out is that #1765 is already adjusting Settings page. So you may want to drop those changes for this PR.

**Comment by carkom** [comment]
Also, same thing I explained to Evo in the other thread. You can't just change colors for this element, because it looks good for this element. All the colors you change are also attached to many other elements across the app.

Here is one example, by changing buttonNormalBackground color, you're effectively changing every button in the app to this color. Which again, might be great for your application but creates issues elsewhere in the app. As one example, you'll see that in the schedules modal below you've changed the cancel button and the date button background color.
image

I'm not saying we can't change these colors I'm just saying that we need to be aware of all the places these changes are effecting the app colors and check all of them. There's a lot more review needed in order to make sure that we are not adding visual regressions unintentionally to other places throughout the app.

**Comment by MatissJanis** [comment]
(a bit of offtopic from this PR)

@carkom perhaps it's time to expand our visual regression tests to darkmode too. We could make it so whenever a lightmode screenshot is captured - also darkmode one is captured and compared. That should give us higher level of reliability for darkmode too.

**Comment by carkom** [comment]
Honestly, I'd focus on sidebar only. If you start changing too many things in one PR it's too hard to keep track of. I think sidebar is in need of an adjustment because it's the same color as the accounts table and as a result looks weird.

Colors Feedback:

1. sidebar background could be slightly lighter (maybe just one shade) IMO.

**Comment by rich-howell** [comment]
image

This could do with some more work, it is really hard to read, it almost looks hazy to me. 

image

Even worse when it is hovered over.

**Comment by rich-howell** [comment]
> image
> 
> This could do with some more work, it is really hard to read, it almost looks hazy to me.
> 
> image
> 
> Even worse when it is hovered over.

If we change this to the same colour used in `--color-tableBorderHover` the AA rules pass.

**Comment by carkom** [comment]
I tried to make sure adjustments to this branch but git is telling me it's been deleted. Closing this PR and opening a new one.

**Pull request closed without merging**